cmd/git_backend.go

```diff
@@ -34,7 +34,7 @@ var gitBackendCmd = &cobra.Command{
 		t, err := template.New(gitHTTPBackendConfigPath).Parse(`
 terraform {
 	backend "http" {
-		address = "{{ .protocol }}://localhost:{{ .port }}/?type=git&repository={{ .repository }}&ref={{ .ref }}&state={{ .state }}"
+		address = "{{ .protocol }}://localhost:{{ .port }}/?type=git&repository={{ .repository }}&ref={{ .ref }}{{ if eq .amend "true" }}&amend=true{{ end }}&state={{ .state }}"
```

Comment:
```suggestion
		address = "{{ .protocol }}://localhost:{{ .port }}/?type=git&repository={{ .repository }}&ref={{ .ref }}&amend={{ .amend }}&state={{ .state }}"
```

Nitpicking but I usually prefer any template to be as plain as possible for readability

Reply:
Thanks @dee-kryvenko , i was going to do it that way too, but noticed it trigger backend change to existing terraform state. Is that okay?

Reply:
anyway, i've updated the templates

Reply:
Ugh.. that's a good catch! Didn't think of that one, that is true. Hate to ask you to revert it back in this case, although uglier but is better from a backward compatibility standpoint.

Reply:
done

Reply:
Thanks!